README.md

```diff
@@ -90,7 +94,9 @@ To learn more about the modern React architecture, we recommend [Making Sense of
 
 The entry point for routing in Waku projects is `./src/entries.tsx`. Export the `createPages` function to create your layouts and pages programatically.
 
-Both `createLayout` and `createPage` accept a configuration object to specify the route path, React component, and render method (`'static'` for SSG or `'dynamic'` for SSR). Layout components must accept a `children` prop.
+Both `createLayout` and `createPage` accept a configuration object to specify the route path, React component, and render method. Waku currently supports two options: `'static'` for static prerendering (SSG) or `'dynamic'` for server-side rendering (SSR).
```

Comment:
What's not describe but technically interesting is that it works without SSR.
In that case, `'static'` means static RSC and `'dynamic'` means dynamic RSC. Even with SSR, the subsequent updates are RSC only and static/dynamic matter.

It's a hard concept and may not fit in README though.